styles/roblox/catppuccin.user.css

```diff
@@ -0,0 +1,199 @@
+    }
+    .btr-explorer-has-children .btr-explorer-more::before, .btr-properties .btr-property-group .btr-property-group-more::before{
+        border-left-color: @accent-color;
+    }
+    .btr-explorer-header .btr-dropdown-container .btr-dropdown-menu > li.selected a{
+       background: @overlay2
+    }
+    .btr-explorer-header .btr-dropdown-container .btr-dropdown-menu > li:hover a{
+        background: @overlay1
+    }
+    .notification, .notification-blue, .notification-red, #notifications-bell-badge.bell-red-badge{
+        background-color: @accent-color;
+        color: @base;
+    }
+    #header .btr-nav-notif{
+        background-color: @accent-color;
+        border-color: @accent-color;
+        color: @base;
+    }
+      .btn-growth-md{
+          background-color: @accent-color !important;
+          border-color: @accent-color !important;
+      }
+      .btn-growth-md:hover, .btn-growth-md:active, .rbx-upgrade-now .btn-secondary-md:hover, .rbx-upgrade-now .btn-secondary-md:active, .btn-primary-lg:hover, .btn-primary-lg:active, .btn-primary-xs:hover, .btn-primary-xs:active, .btn-common-play-game-lg:active, .btn-common-play-game-lg:hover, .btn-growth-xs:hover, .btn-growth-xs:active, .btn-growth-sm:hover, .btn-growth-sm:active, .btn-growth-lg:active, .btn-growth-lg:hover, .btn-primary-md:focus{
+          background-color: @accent-color !important;
+          border-color: @accent-color !important;
+          filter: brightness(83%)
+      }
+      .rbx-upgrade-now .btn-secondary-md, .btn-primary-md, .btn-primary-lg, .btn-primary-xs, .btn-common-play-game-lg, .btn-growth-xs, .btn-growth-sm{
+          background-color: @accent-color;
+          border-color: @accent-color;
+      }
+      .shopping-cart-btn-container .shopping-cart-btn .shopping-cart-icon-container .cart-number-container, .pill-toggle input:checked+label, .radio input[type=radio]:checked+label:after, .btr-switch .btr-switch-flip{
+          background: @accent-color;
+      }
+      .btn-growth-lg{
+          background: @accent-color !important;
+          border-color: @accent-color !important;
+      }
+      .btr-toggle-description, .btr-fastsearch-placename{
+          color: @accent-color !important;
+      }
+      
+
+      #randomGameButton{
+          background: transparent !important;
+      }
+      .menu-vertical .menu-option .menu-option-content.active, .menu-vertical .menu-option .menu-option-content:active, .menu-vertical .menu-option .menu-option-content:focus, .menu-vertical .menu-option .menu-option-content:hover{
+          box-shadow: inset 4px 0 0 0 @accent-color;
+      }
+      #navbar-search-input, .input-field{
+          background-color: @base !important;
+      }
+      .dialogs .dialog-container .dialog-message-container.message-inbound .dialog-triangle:after, .dialogs .dialog-container .dialog-message-container.message-inbound .dialog-triangle:before{
+          border-color: transparent @mantle transparent transparent;
```

Comment:
Isn't this the equivalent of `border-bottom-color: @mantle`?

Reply:
I'll edit that rn

Reply:
I believe so. I think I was just following how Roblox did it and they did ``border-color: transparent [color] transparent transparent`` Will change it